src/helpers/checkArgs.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable node/prefer-global/process */
+import { BG_ERR, BG_RESET } from '../rules/asceeCodes'
+
+const checkArgs = (argv: string[], config: { apply?: any, ignore?: any }) => {
+  const hasIgnore = argv.some(arg => arg.startsWith('--ignore') || arg.startsWith('-i'))
+  const hasApply = argv.some(arg => arg.startsWith('--apply') || arg.startsWith('-a'))
```

Comment:
I think the purpose of these lines is the same as this helper function `wasOptionPassed` from `/helpers/wasOptionPassed.ts`?

```ts
// Helper function to determine if an option was passed via CLI
export function wasOptionPassed(option: string) {
  // Regular expression to match the exact option with any value
  const regex = new RegExp(`--${option}(?:=[^\\s]*)?$`)

  // eslint-disable-next-line node/prefer-global/process
  return process.argv.some(arg => regex.test(arg))
}

```

If you think it doesnt apply, that helper function is not being used anymore so it should be deleted